main.py

```diff
@@ -217,7 +253,7 @@ async def logout_endpoint(id_token: str = Cookie(""), access_token: str = Cookie
     await revoke_token(access_token)
 
     # prepare the redirection response
-    response = RedirectResponse(CONFIG["url_redirect"])
+    response = RedirectResponse("/")
```

Comment:
The `/logout` endpoint is used as an API to redirect the user to perform a logout at the AAI. It's used for example in Beacon Network, when the user clicks the `Log Out` button, they are redirected to LS AAI to end their session, and then back to Beacon Network.

With this change it becomes a GUI, the user would be redirected to the Auth API to click another `logout` link, and then not be redirected back to the original website.

Reply:
Oh, is tiny-rp a component in Beacon Network UI?

Reply:
Yes, it was originally created for it as a quick replacement for an old auth service that stopped working due to some package bugs :smile:

The old Beacon Network is expected to be closed this year :crossed_fingers: so after that it won't be much an issue there. But after that it will still be in used in BIGPICTURE Beacon Network.

Reply:
I see, nice to have the context!